content/md/first-steps/manage-your-attributes.md

```diff
@@ -96,15 +52,20 @@ To create an attribute in the PIM:
     - **A value per locale** (Yes/No): Yes if your attribute values depend on the locales
     - **Locale specific**: if it must apply to specific locales not all of them
     - **Usable in grid**: whether or not the attribute can be displayed as a column or used as a filter in the product grid or in the proposal grid (EE only - since 2.2)
+    - **Read only**: only for Enterprise Edition users, when enabled it prevents this attribute from being edited in the UI, it can only be updated through imports or rules.
     - **Validation parameters**: depending on the attribute type chosen (see [Validation for the different attribute types](#validation-for-the-different-attribute-types))
 1. Then click on `Save`
 
-::: ee
-The property "read only" prevents this attribute from being edited in the UI, it could be updated only through imports or rules.
-:::
-
-The attribute has been created and some properties can be added or amended over time.
+The attribute is now created, if you need to edit an attribute property, please refer to the next section [Edit attribute properties] (/articles/manage-your-attributes.html#edit-attribute-properties)
```

Comment:
Remove the space between the part into brackets and the part into parenthesis if you want the link to actually work.